feature-libs/checkout/components/components/review-submit/review-submit.component.html

```diff
@@ -175,16 +175,16 @@ <h2 class="cx-review-title d-none d-lg-block d-xl-block">
 
   <!-- CART ITEM SECTION -->
   <ng-container *ngIf="cart$ | async as cart">
-    <h4 class="cx-review-cart-total d-none d-lg-block d-xl-block">
+    <div class="cx-review-cart-total d-none d-lg-block d-xl-block">
```

Comment:
Should be transferred to both base and b2b templates.

Reply:
don't worry! as explained here https://sap-cx.slack.com/archives/D02A0NHD3C0/p1643298314004500?thread_ts=1643268177.004900&cid=D02A0NHD3C0